plugins/httpapi/nd.py

```diff
@@ -85,6 +85,24 @@ def set_backup_hosts(self):
         except Exception:
             return []
 
+    # Required for cisco.dcnm modules
+    # The cisco.nd HTTPAPI provider only supports NDFC 12+
+    # TODO Add support for more platforms.
+    # TODO Add support for dynamically returning platform versions.
+    def get_version(self, platform="ndfc"):
+        if platform == "ndfc":
```

Comment:
Do we want to define this as a constant, type dictionary? Then we could use a `return PLATFORMS.get(platform)`. 

I am not familiar with the flow of DCNM modules but why return None instead of raising a undefined platform type error?

Reply:
I kept it simple for now to keep compatibility with the `ansible-dcnm` modules. We may need to enhance and use this function in the future for newer ND versions.
It will now throw a value error if an unsupported platform is used.